packages/fx-core/resource/package.nls.json

```diff
@@ -1034,5 +1034,9 @@
   "error.yaml.YamlFieldTypeError": "Field '%s' should have %s type, yaml file: %s",
   "error.yaml.YamlFieldMissingError": "Field '%s' is missing, yaml file: %s",
   "error.yaml.InvalidYmlActionNameError": "Action '%s' not found, yaml file: %s",
-  "error.yaml.LifeCycleUndefinedError": "Lifecycle '%s' is undefined, yaml file: %s"
+  "error.yaml.LifeCycleUndefinedError": "Lifecycle '%s' is undefined, yaml file: %s",
+  "error.common.UnresolvedPlaceholderError": "There are unresolved placeholders: '%s', which are required for file: %s.",
```

Comment:
Because the placeholder is actually environment variable, can we directly tell users xxx environment variable is missing? So users do not need to map 'placeholder' to 'environment variable' when reading the error.

Reply:
The final strings can be reviewed and updated in another PR

Reply:
Thanks, I'm OK to do it in another PR.